api/src/opentrons/protocol_engine/commands/move_labware.py

```diff
@@ -226,11 +226,10 @@ async def execute(self, params: MoveLabwareParams) -> _ExecuteReturn:  # noqa: C
                     y=0,
                     z=0,
                 )
-                eventual_destination_location_sequence = [
-                    NotOnDeckLocationSequenceComponent(
-                        logicalLocationName=OFF_DECK_LOCATION
-                    )
-                ]
+                state_update.set_addressable_area_used(
+                    addressable_area_name=area_name,
+                )
+
```

Comment:
In the original version of this, we set the eventual destination location to `OFF_DECK_LOCATION`. You want to do the same here but with your new type. Should be something like:
```suggestion
                eventual_destination_location_sequence = [
                    NotOnDeckLocationSequenceComponent(
                        logicalLocationName=WASTE_CHUTE_LOCATION
                    )

```
Setting the addressable area as used can be removed here, because it is already occurring up on line 213. Setting an addressable area as used doesn't actually mean it's occupied, it just signals our state tracking that "this addressable area must be in the deck configuration". In the case of the waste chute, it tells us that we cannot have a covered waste chute in the deck config, because that won't provide a hole to drop labware through.